packages/markdown-render-engine/src/renderers/ListItem.tsx

```diff
@@ -1,4 +1,5 @@
 import React from "react";
+import { createElementClass } from "../helpers";
 
 export interface ListItemProps {
   checked: boolean | null;
```

Comment:
wouldn't `boolean` be enough? as you have two falsy values in this union now

Reply:
Hmm, interesting. Props for renderers is copied from old implementation of our markdown renderer in website, co probably this is wrong, thanks :)

Reply:
`null` could have been used as some kind of initial value, idk

better change it